### DISCUSSION TITLE
slow speed with nuxt/image in nuxt 3

### DISCUSSION THREAD

**Discussion by Ali-0111** [nuxt/nuxt#25999] [Questions]
In my project, I utilized NuxtImg to optimize the images. The original image I worked with was in "jpg" format and had a file size of 1.5 megabytes (Mb). By leveraging NuxtImg, I was able to convert the image format to WebP with a width of 300 pixels and a height of 200 pixels. As a result, the converted image size significantly decreased to just 7.1 kilobytes (kb), while the corresponding placeholder image was only 455 bytes (B) in size. This reduction in file size is impressive and beneficial for optimizing page load times. However, it is worth noting that the process of generating the placeholder image  and retrieving the actual image took more than 10 seconds .
Additionally, I encountered a similar issue with different sources while using images in my project. Despite using various CDNs, I faced the same problem. 

Therefore, what do you recommend me to find essential solution for time issue?
Original image: 1.5Mb, jpg, 

# Here is the result:
![heroIssue](https://github.com/nuxt/nuxt/assets/73581663/98ad1ade-e4cb-4de4-a9b9-ab43a12ae18e)

**Comment by szobX** [comment]
any update with this "issue"?
